openid-federation-1_0.xml

```diff
@@ -6942,8 +6947,8 @@ HTTP/1.1 302 Found
 	</t>
 	<t>
 	  For instance, an enterprise application might use
-	  the <spanx style="verb">id</spanx> (identifier) claim
-	  to convey an employee ID value.
+	  the <spanx style="verb">trust_mark_id</spanx> (identifier) claim
+	  to convey an ID value.
```

Comment:
Please put back in "employee" or another suitable example to illustrate what the ID could be about.